src/rust/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [4.1.0] - 2024-02-24
+
+- Potentially reduce requests for feature data by correctly including distance
+  to the next feature in the index traversal in all cases.
+- Fix HTTP 416 on webservers that don't support Range requests that extend
+  beyond the end of the file. This was only applicable to bbox requests.
+- Upgrade to geozero 0.12.0
+- Make HttpFgbReader generic
```

Comment:
If I understood correctly, this was not a breaking change, right @kylebarron ?

Reply:
To my understanding it was not a breaking change